.gitignore

```diff
@@ -1,3 +1,5 @@
 *.o
 qdl
 ks
+compile_commands.json
```

Comment:
What is LSP?

Reply:
I see. A purist in me says that this is irrelevant to the ramdump, but who cares :-D

Reply:
Ahh, you're right. I forgot that we (typically) end up with a merge commit when accepting the PR. Too long since I contributed outside the kernel...